Project.toml

```
@@ -43,7 +43,7 @@ MacroTools = "0.5.1"
 PDMats = "0.9.9"
 ProgressMeter = "1.0.0"
 Reexport = "0.2.0"
-Requires = "0.5.2"
+Requires = "0.5.2, 1.0"
```

```suggestion
Requires = "1.0"
```

Does anyone see a reason not to drop 0.5.2?